.envrc

```diff
@@ -35,6 +35,13 @@ export SAML_ISSUER=ussf-portal-client
 export SESSION_SECRET=thisvaluecanbeanythingitisonlyforlocaldevelopment
 export SESSION_DOMAIN=localhost
 
+
+##############################
+# Keystone Configuration     #
+##############################
+export NEXT_PUBLIC_KEYSTONE_URL=http://localhost:3001
```

Comment:
If we are actually only using the Keystone URL from the server, does this still need to be a `NEXT_PUBLIC...` var? My understanding of those is that they get baked in if they are defined at build time and can be a little confusing, so if we aren't using this in any browser code then it might be best to rename to just `KEYSTONE_URL`.

Reply:
will we only ever use keystone on the server?

Reply:
we probably will end up referencing it in both server & browser. I'm really just suspicious of `NEXT_PUBLIC_` because that was how I had tried to define other env vars (like the Matomo URL) and there were issues when deploying 🤷‍♀️